app/helpers/episodes_helper.rb

```diff
@@ -39,8 +39,8 @@ def episodes_path_or_podcast_episodes_path(podcast_id = nil)
     end
   end
 
-  def episode_content_duration(content)
-    Time.at(content.duration || 0).utc.strftime("%H:%M:%S").sub(/^00:/, "0:")
+  def episode_media_duration(media)
```

Comment:
One theme throughout this PR ... I'm renaming "audio" and "content" terminology to just be "media".

Because: episodes can be video.  Or uncut audio.  And it's less confusing to just refer to the parent model `media_resource.rb` (or the episode concern `episode_media.rb`) instead.